### PR TITLE
Add 404 page

### PR DESCRIPTION
**Pull request by sandylcruz**
* Next documentation on creating a 404 page [here](https://nextjs.org/docs/advanced-features/custom-error-page)
* Preview [here](https://blog-git-sandy-404-gravitational.vercel.app/blog/404/)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202002821619688

### CODE REVIEW

File: pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+const StyledLink = styled(Link)(
+  css({
+    color: "light-blue",
+    "&:visited": {
+      color: "light-purple",
+    },
+  })
+);
+
+const Title = styled("h1")(
```
Comment (deliaconstantino) [inline pages/404.tsx:62]: should this be a `styled("h2")` instead? I believe for SEO purposes we should aim for one h1 per page.

File: pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  css({
+    fontSize: ["26px", "header-1"],
```
Comment (deliaconstantino) [inline pages/404.tsx:64]: should this use the closest option from the theme rather than "26px"?

Comment (sandylcruz) [reply pages/404.tsx]: updated to "header-3"

**Review by deliaconstantino — APPROVED** [review]

File: pages/404.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    name: "About Us",
+    href: "https://goteleport.com/about",
+  },
+  {
+    name: "Documentation",
+    href: "https://goteleport.com/docs",
+  },
+  {
+    name: "Blog",
+    href: "/",
```
Comment (C-STYR) [inline pages/404.tsx:86]: Let's make this an absolute URL please - using `/` has caused problems elsewhere.

**Review by C-STYR — COMMENTED** [review]
Just one update please @sandylcruz

**Comment by C-STYR** [comment]
Actually, can you please add a trailing slash to all of the others as well...

**Pull request merged**